elements/lisk-validator/src/formats.ts

```diff
@@ -125,3 +127,41 @@ const camelCaseRegex = /^[a-z]+((\d)|([A-Z0-9][a-zA-Z0-9]+))*([a-z0-9A-Z])?$/;
 
 export const camelCase = (data: string): boolean =>
 	camelCaseRegex.exec(data) !== null;
+
+export const version = (data: string): boolean =>
+	/^([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3})(-(alpha|beta|rc)\.[0-9]{1,3}(\.[0-9]{1,3})?)?$/.test(
+		data,
+	);
+
+export const protocolVersion = (data: string): boolean =>
+	/^(\d|[1-9]\d{1,2})\.(\d|[1-9]\d{1,2})$/.test(data);
+
+export const path = (data: string): boolean => {
+	const pathRegExp = new RegExp('^(.?)(/[^/]+)+$');
```

Comment:
Unescaped forward slash in this regex. 

<img width="192" alt="image" src="https://user-images.githubusercontent.com/112468/84250323-31b31a80-ab0c-11ea-97af-bf36866c54dc.png">

This does not work with paths having a forward slash at the end, which is quite common for a directory path. 

```
/nazar/alpha/beta/
```